broker-daemon/broker-rpc/wallet-service/commit.js

```diff
@@ -67,14 +61,22 @@ async function commit ({ params, relayer, logger, engines, orderbooks }, { Empty
     throw new PublicError(`Minimum balance of ${MINIMUM_FUNDING_AMOUNT} needed to commit to the relayer`)
   } else if (maxChannelBalance.lt(balance)) {
     logger.error(`Balance from the client exceeds maximum balance allowed (${maxChannelBalance.toString()}).`, { balance })
-    throw new PublicError(`Maximum balance of ${maxChannelBalance.toString()} exceeded for committing of ${balance} to the relayer. Please try again.`)
+    throw new PublicError(`Maximum balance of ${Big(maxChannelBalance).div(engine.currencyConfig.quantumsPerCommon).toString()} ${symbol} exceeded for committing of ${balanceInCommonUnits} ${symbol} to the Relayer. Please try again.`)
```

Comment:
These messages would be much cleaner if you defined `engine.currencyConfig.quantumsPerCommon` etc to variables.